ubuntucleaner/janitor/deb_installers_plugin.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import os
+
+from ubuntucleaner.janitor import JanitorPlugin
+from ubuntucleaner.settings.debug import get_traceback, run_traceback
+from ubuntucleaner.janitor import CruftObject
+
+log = logging.getLogger('Janitor')
+
+class DebInstallersPlugin(JanitorPlugin):
+    __title__ = _('Deb Installers')
+    __category__ = 'personal'
+
+    def __init__(self):
+        self.downloads_folder = os.path.expanduser("~/Downloads")
+        JanitorPlugin.__init__(self)
+
+    def get_cruft(self):
+        try:
+            count = 0
+            size = 0
+            for file in os.listdir(self.downloads_folder):
+                if file.endswith(".deb"):
```

Comment:
Instead you could extend from JanitorCachePlugin and define self.pattern as '.deb' so the function get_cruft_by_glob will be triggered (see how `get_cruft` is implemented)